rclpy/topics/pointcloud_publisher/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>pointcloud_publisher</name>
```

Comment:
To be more consistent with the other packages, this should be called something like `examples_rclpy_pointcloud_publisher`.